cmd/config.go

```diff
@@ -62,17 +78,28 @@ func getAuthentication(flags *pflag.FlagSet) (settings.AuthMethod, auth.Auther)
 
 	if method == auth.MethodJSONAuth {
 		jsonAuth := &auth.JSONAuth{}
-
 		host := mustGetString(flags, "recaptcha.host")
 		key := mustGetString(flags, "recaptcha.key")
 		secret := mustGetString(flags, "recaptcha.secret")
 
-		if key != "" && secret != "" {
-			jsonAuth.ReCaptcha = &auth.ReCaptcha{
-				Host:   host,
-				Key:    key,
-				Secret: secret,
-			}
+		if key == "" {
+			kmap := defaultAuther["recaptcha"].(map[string]interface{})
+			key = kmap["key"].(string)
+		}
+
+		if secret == "" {
+			smap := defaultAuther["recaptcha"].(map[string]interface{})
+			secret = smap["secret"].(string)
+		}
+
+		if key == "" || secret == "" {
+			panic(nerrors.New("you must set the flag 'recaptcha.key' and 'recaptcha.secret' for method 'json'"))
```

Comment:
Please use `checkErr`

Reply:
I did not get how to use checkErr in this block though.

Reply:
Just replace `panic` by `checkErr`

Reply:
done